pages/linux/pkgctl-db-update.md

```diff
@@ -0,0 +1,8 @@
+# pkgctl db update
+
+> Update the pacman database as final release step for packages that have been transfered and staged on <repos.archlinux.org>.
+> More information: <https://man.archlinux.org/man/pkgctl-build.1>.
```

Comment:
Wrong man page.
```suggestion
> More information: <https://man.archlinux.org/man/pkgctl-db-update.1>.
```

Reply:
Nice catch, will apply the suggestions. Thanks.